embc-app/Migrations/20190426215123_Country-PK-Change.cs

```diff
@@ -284,6 +284,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
                                                 UPDATE Countries SET CountryCode = 'VNM' WHERE Name = 'Viet Nam'
                                                 UPDATE Countries SET Name = 'Virgin Islands (British)' WHERE Name = 'Virgin Islands, British'
                                                 UPDATE Countries SET CountryCode = 'VGB' WHERE Name = 'Virgin Islands (British)'
+                                                UPDATE Countries SET Name = 'Virgin Islands (U.S.)' WHERE Name = 'Virgin Islands, U.S.'
```

Comment:
I'd prefer to generate the update from the old json file to ensure the names are the same

Reply:
Me as well.  Just couldn't figure out a clean way to inject the DataInterface.